repos/spack_repo/builtin/packages/geant3/package.py

```diff
@@ -39,7 +41,7 @@ class Geant3(CMakePackage):
     def cmake_args(self):
         args = []
         if self.spec.satisfies("%gcc@10:"):
-            args.append('-DCMAKE_Fortran_FLAGS="-fallow-argument-mismatch -fallow-invalid-boz"')
+            args.append("-DCMAKE_Fortran_FLAGS=-fallow-argument-mismatch -fallow-invalid-boz")
```

Comment:
Even better (and avoiding quote issues) is to use the `self.define` helper:
```suggestion
            args.append(self.define("CMAKE_Fortran_FLAGS", "-fallow-argument-mismatch -fallow-invalid-boz"))
```

Reply:
Ah even better, thanks!